### ISSUE TITLE
Doesn't produce anything close to working code for example "cli snake game" given in readme

### ISSUE THREAD

**Issue by Benzidrine** [geekan/MetaGPT#195]
Cool idea. Thanks for creating it. Interesting to try it out. However the example given in the readme doesn't lead to working code. Unfortunately it tries to instantiate classes without providing necessary properties.

Using chatgpt4 and code review. Unfortunately it doesn't seem suitable for any task.

Note I don't think it is necessarily your fault. ChatGPT4 I think has gotten much worse.

**Comment by voidking** [comment]
Yes, we cannot guarantee that the generated code will work, it relies heavily on gpt. There is some work in the roadmap that may improve the success rate of the first run.
